docusaurus/docs/Android/04-guides/04-theming.md

```diff
@@ -1,4 +1,8 @@
-# Global Styling
+# Theming
+
+Many parts of the UI components can be changed and customized. It is possible to change colors, fonts, sizes and many other aspects of the `Views` to match a desired look. 
```

Comment:
It would be great to expand this description. Maybe we can write that currently views can be customized either by attributes or programmatically by TransformStyle class and then describe all the things that suppose to be customizable and which are not. For example - we don't support changing drawables tint but we do support changing drawables.